test/v2/models/responses/hateoas/HateosMovementResponseSpec.scala

```diff
@@ -24,20 +24,20 @@ import play.api.libs.json.Json
 import v2.models.EORINumber
 import v2.models.MovementId
 import v2.models.MovementReferenceNumber
-import v2.models.responses.DepartureResponse
+import v2.models.responses.MovementResponse
 
 import java.time.OffsetDateTime
 import java.time.ZoneOffset
 
-class HateosDepartureResponseSpec extends AnyFreeSpec with Matchers with ScalaCheckDrivenPropertyChecks {
+class HateosMovementResponseSpec extends AnyFreeSpec with Matchers with ScalaCheckDrivenPropertyChecks {
 
   private val hexId    = Gen.listOfN(16, Gen.hexChar).map(_.mkString.toLowerCase)
```

Comment:
This can be moved to CommonGenerators

Reply:
Yes, you can use arbitraryMovementId in CommonGenerators, which generates the hexId and creates the object

Reply:
yes, we can use arbitraryMovementId